### PR TITLE
Add link to training, clarify "Need Help" section

### PR DESCRIPTION
**Pull request by tim-o**
Made "Need help" consistent with the [github template PR](https://github.com/cockroachdb/cockroach/pull/27811), and added references to the 2.0.4 training. @jseldess - are you OK with us pushing folks to the material?

If there's anyone else who should review a change to the readme, let me know.

### CODE REVIEW

File: README.md
Comment (jseldess) [inline README.md]: nit: `self guided training` > `self-guided training`

**Review by jseldess — APPROVED** [review]
LGTM, with a few nits. Thanks, @tim-o!

File: README.md
Comment (jseldess) [inline README.md]: nit: `first time` > `first-time`

File: README.md
Comment (jseldess) [inline README.md]: Missing the closing bracket after `Forum`.

File: README.md
Comment (jseldess) [inline README.md]: Missing period at end of sentence.

**Comment by jseldess** [comment]
@tim-o, can you address my comments before merging?

**Comment by tim-o** [comment]
Of course, I thought that was taken care of - I'm sorry, this is like PR #2, I'm still not super familiar with github.

**Comment by tim-o** [comment]
@jseldess - updated 
 here: https://github.com/cockroachdb/cockroach/pull/27813/commits/94399e6f2db9962f5ba57be5027e540dc917636e.

**Review by jseldess — APPROVED** [review]
LGTM. Thanks, @tim-o.

**Comment by jseldess** [comment]
@tim-o, do you still want to merge this?

**Comment by jseldess** [comment]
@tim-o, ping.

**Comment by tim-o** [comment]
@jseldess Removed conflicts, just waiting on Borse and CI and I'll merge.

**Comment by jordanlewis** [comment]
@jseldess do you still want this, should we close?

**Comment by jseldess** [comment]
@jordanlewis, I think we should close this at this point, as some of it is no longer accurate. I'd love to open a new PR with recommended readme updates, though.

**Pull request closed without merging**
